js/src/components/merchant-center-select-control/index.js

```diff
@@ -27,10 +37,13 @@ const MerchantCenterSelectControl = ( props ) => {
 	options.sort( ( a, b ) => {
 		return a.label.localeCompare( b.label );
 	} );
-	options.unshift( {
-		value: '',
-		label: __( 'Select one', 'google-listings-and-ads' ),
-	} );
+
+	useEffect( () => {
+		// Triggers the onChange event in order to pre-select the initial value
+		if ( value === undefined ) {
+			onChange( options[ 0 ].value );
+		}
+	}, [ options, onChange, value ] );
 
 	return <AppSelectControl options={ options } { ...props } />;
```

Comment:
What do you think if we do the following to add the default value:

```suggestion
	return		<AppSelectControl
			options={ options }
			{ ...props }
			value={
				props.value === undefined ? options[ 0 ].value : props.value
			}
		/>;
```

And it does not need the `useEffect` neither to extract the props.